service/src/main/java/de/adorsys/psd2/sandbox/portal/testdata/TestDataService.java

```diff
@@ -458,11 +494,79 @@ private PsuData initPsuPending() {
         "Amazon.de: Ihre Bestellung #4528499"
     );
 
-    return new PsuData(
+    return new TestPsu(
+        "PSU-Pending",
+        GLOBAL_PASSWORD,
+        GLOBAL_TAN,
+        initSingleAccount(accountId, iban, BigDecimal.valueOf(9.21), transaction),
+        "Pending",
+        "valid",
+        "finalised",
+        null,
+        null,
+        "finalised"
+    );
+  }
+
+  private TestPsu initPsuConsentExpired() {
+    String iban = "DE86760300801729983660";
+    String accountOwner = "Meryl Streep";
+    String accountId = "82d10b08-9d41-4211-9e80-130a892a4d8f";
+
+    Transaction transaction = new Transaction(
+        "3bf5b19b-12e9-4ab4-bdac-a5c7695ae4b9",
+        BigDecimal.valueOf(199.99),
+        EUR,
+        LocalDate.parse("2018-09-20"),
+        accountOwner,
+        iban,
+        "Amazon",
+        "",
+        "Amazon.de: Ihre Bestellung #4528499"
+    );
+
+    return new TestPsu(
+        "PSU-Pending",
+        GLOBAL_PASSWORD,
+        GLOBAL_TAN,
+        initSingleAccount(accountId, iban, BigDecimal.valueOf(9.21), transaction),
+        "AcceptedCustomerProfile",
+        "expired",
+        "finalised",
+        "Canceled",
+        null,
+        "finalised"
+    );
+  }
+
+  private TestPsu initPsuConsentRevokedByPsu() {
+    String iban = "DE86760300801729983660";
+    String accountOwner = "Tom Cruise";
```

Comment:
I would use a more neutral and generic name.

Reply:
We will change the names and transactions in a following separate commit.